pkg/process/cache.go

```diff
@@ -132,7 +131,7 @@ func NewCache(
 	lruCache, err := lru.NewWithEvict(
 		processCacheSize,
 		func(_ string, _ *ProcessInternal) {
-			mapmetrics.MapDropInc("processLru")
+			errormetrics.ErrorTotalInc(errormetrics.ProcessCacheEvicted)
```

Comment:
but in both cases evictions can happen right ?

Reply:
My understanding is that the eviction callback (defined here) is called on every eviction, so incrementing the metric in the `add` method too duplicates it. I would increment the metric only in the eviction callback. cc @willfindlay to confirm.

Reply:
ok by looking at the code from first point I feel whenever we initiate the cache with NewCache with some item and then maybe add other items using ```add``` so both cases eviction can happen . so that's where I thought both place we might consider to add this condition . let me know if I am wrong

Reply:
@lambdanis is correct. We should remove the other increment and only count it here.